### DISCUSSION TITLE
How to get values from package.json from compiled code?

### DISCUSSION THREAD

**Discussion by studiojms** [swc-project/swc#9456] [Q&A]
I'm migrating one of my projects using NestJS to use vitest. One of the requirements for vitest is to add swc as the compiler to my NestJS project.
In my codebase, there are some code relying on the `version` and `name` declared in `package.json`, but using swc the package.json file is not available in the `dist` folder.

Any ideas on how to proceed to get the name and version props from package.json in the compiled code using swc?

I tried `process.env.npm_package_name` and `process.env.npm_package_version`, but they don't work in the compiled code as well.

**Comment by kdy1** [comment]
Try --copy-files
